sanssouci/lambda_calibration.py

```diff
@@ -220,25 +221,43 @@ def calibrate_jer(alpha, learned_templates, pval0, k_max, min_dist=3):
     k_max : int
         template size
     min_dist : int
-        minimum distance to stop iterating dichotomy
+        minimum distance to stop iterating dichotomy. Default = 1.
 
     Returns
     -------
 
-    int : index of template chosen by calibration
+    thr : list of length k_max
+        Threshold family chosen by calibration
 
     """
 
+    # Sort permuted p-values
+    pval0 = np.sort(pval0, axis=1)
+
     B, p = learned_templates.shape
     low, high = 0, B - 1
+
+    if estimate_jer(learned_templates[high], pval0, k_max) <= alpha:
```

Comment:
Raise a warning here too? If the largest family has a JER less than alpha the control will possibly be conservative. I don't see a good reason why this should happen (which seems to be an argument in favor of raising a warning)